tests/src/main/java/net/neoforged/neoforge/oldtest/DisallowItemPickupTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.oldtest;
+
+import net.neoforged.fml.common.Mod;
+import net.neoforged.neoforge.common.NeoForge;
+import net.neoforged.neoforge.common.Tags;
+import net.neoforged.neoforge.event.entity.item.ItemAllowPickupEvent;
+
+/**
+ * Basic test mod to test the new {@linkplain ItemAllowPickupEvent}.
+ * <p>
+ * When this test is enabled the following should be true
+ * <ul>
+ * <li>Items ontop of {@linkplain Tags.Blocks#STONES} should not be allowed to be picked up</li>
+ * </ul>
+ */
+@Mod(DisallowItemPickupTest.ID)
+public final class DisallowItemPickupTest {
```

Comment:
Please convert this to a game test.

Reply:
I have never really used game tests or neos test frame work before but I believe I have converted the test correctly